sycl/tools/sycl-vxx/bin/sycl_vxx.py

```diff
@@ -144,8 +144,7 @@ def _run_optimisation(self):
                 "-flat-address-space=0", "-globaldce"
             ])
         opt_options.extend([
-            "--vectorize-loops=false", "--disable-loop-unrolling", "-O3",
-            "-globaldce", "-globaldce", "-inSPIRation",
+            "-inSPIRation",
```

Comment:
All these should be commented.
Otherwise here we struggle to understand why you just removed in `_run_optimisation` all the optimizations :-)